source/contracts/libraries/Extractable.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity 0.4.17;
+
+import 'libraries/token/ERC20Basic.sol';
+import 'Controlled.sol';
+
+
+contract Extractable is Controlled {
+    // Send accidentally received ETH to the destination
+    function extractETH(address _destination) public onlyControllerCaller returns(bool) {
```

Comment:
Style: `Eth` (lower case last two letters)

Reply:
Consider just calling it `Ether` in function/variable names.  I wonder if it would be good to just use `Ether` everywhere in our codebase rather than `ETH`/`Eth`/`eth`?  We aren't really saving anything by abbreviating in this case.

Reply:
Will make an issue to do this elsewhere at some point